dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.6/com.ibm.websphere.appserver.org.eclipse.persistence-2.6.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.6
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink; apiJar=false; location:=dev/api/third-party/, \
```

Comment:
we will also want to provision `-feature=com.ibm.websphere.appserver.javax.persistence.base-2.1` here